orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -129,24 +129,31 @@ var OrcidCookie = new function() {
     };
 };
 
-var _gaq = _gaq || [];
-
 var OrcidGA = function() {
     // test and make sure _gaq is working. disconnect.me chrome plugin has
     // caused silent _gaq failures. This check allows us to detect that
     // situation
     var gaEnabled = false;
-    _gaq.push(function() {
-        gaEnabled = true;
-    });
-
+    if(window.ga && ga.create) {
+    	gaEnabled = true;
+    }
+    
     this.buildClientString = function(clientGroupName, clientName) {
         return clientGroupName + ' - ' + clientName
     };
     this.gaPush = function(trackArray) {
         if (gaEnabled) {
-            _gaq.push(trackArray);
-            console.log("_gap.push for " + trackArray);
+        	if(typeof trackArray === 'function') {
+        		ga(trackArray);
```

Comment:
I changed this line, so, it uses the ga queue so it executes the callback in the order it was called by the app.